index.js

```diff
@@ -1,18 +1,42 @@
 /***** Deliverable 1 *****/
-const header = document.querySelector("#header")
+const header = document.querySelector("h1#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
+// debugger
+header.style.color = "green"
 
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
 
+const travelerName = document.querySelector('h2')
+const travelerNickname = document.querySelector('em')
+const travelerPhoto = document.querySelector('img')
+
+travelerName.textcontent = traveler.name
+travelerNickname.textcontent = traveler.nickname
+travelerPhoto.src = traveler.photo
 
 /***** Deliverable 4 *****/
 
+traveler.animalSightings.forEach(function(object){
+    const outerDiv = document.createElement('li')
+
+    outerDiv.innerHTML = `
+    <li data-id="${object.id}">
+    <p>"${object.description}"</p>
+    <img src="${object.photo}" alt="${object.species}"/>
+    <a href="${object.link}" target="_blank">"Here's a video about the ${object.species} species!"</a>
+    </li>`
 
+    const collection = document.querySelector('#animals')
+    collection.append(outerDiv)
+
+})
```

Comment:
good. Be careful with variable names.

Would prefer the outerDiv to be renamed to animalSightingsLI because it is an LI and not a div.

since you are using innerHTML for the LI. You are actually nesting another LI inside of it here. You can set the data-id attribute before setting the innerHTML.

```
traveler.animalSightings.forEach(obj => {
    const animalSightingsLI = document.createElement('li')
    animalSightingsLI.dataset.id = obj.id

    animalSightingsLI.innerHTML = `
        <p>${obj.description}</p>
        <img src="${obj.photo}" alt="${obj.species}"/>
        <a href="${obj.link}" target="_blank">Here's a video about the ${obj.species} species!</a>
    `
    const animalsUL = document.querySelector('ul#animals')
     animalsUL.append(animalSightingsLI)
})
```